# Today I learned...
======
#### 2016-11-18: Java Finally block is still executed when you throw an exception in the Catch.
